src/cert_context.rs

```diff
@@ -76,19 +86,27 @@ impl CertContext {
         }
     }
 
-    /// Returns the sha1 hash of this certificate
-    ///
-    /// The sha1 is returned as a 20-byte array representing the bits of the
-    /// sha1 hash.
-    pub fn sha1(&self) -> io::Result<[u8; 20]> {
```

Comment:
Right. I actually did want to add it as wrapper for fingerprint, seems like I didn't.
Anyways fixed now.